VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.ts

```diff
@@ -145,10 +145,10 @@ export class JudgeWaitingRoomComponent extends WaitingRoomBaseDirective implemen
             )
             .subscribe(createdParticipant => {
                 const participantDisplayName = PexipDisplayNameModel.fromString(createdParticipant.pexipDisplayName);
-                this.participantRemoteMuteStoreService.assignPexipId(participantDisplayName.participantOrVmrId, createdParticipant.uuid);
+                this.participantRemoteMuteStoreService.assignPexipId(participantDisplayName?.participantOrVmrId, createdParticipant.uuid);
```

Comment:
Any potential issues calling assignPexipId with a null value here?

Reply:
As discussed, this line is to avoid console log error, there is subsequent PR for VIH-8587 and this code is refactored to handle the above. thanks